### DISCUSSION TITLE
New Site. New Astroid Templates give500 error

### DISCUSSION THREAD

**Discussion by srcarney** [templaza/astroid-framework#850] [Q&A] [answered]
I just installed a new Joomla install in a temporary development site at https://shalomd1.jdevcloud.com/

If I select the Cassiopei template style a page displays. 

I also installed the three templates from astroidframe.work

But, if I try to view those I see a 500 error. 

Sorry, there was a problem we could not recover from.

The server returned a "500 - Whoops, looks like something went wrong."

I am wondering if you can tell me what I may have done wrong. 

Also, I did try to install the quickstart in another development site but was not able to login because I did not know the admin login and password. 

Again, I am probably doing something wrong. Any advice will be appreciated.

Joomla version 5.2.0
PHP 8.3.11

Sean Carney

**Comment by Banuk1616** [comment, marked as answer]
Have you checked whether the Astroid plugin is activated?

The user name for the Quicksart should be ‘admin’, you set the password yourself during installation.

Best Regards

**Reply by srcarney** [reply]
The Astroid plugin was not even installed. It is now and all is well. Thanks. I had assumed the framework would install with the templates.
